In [30]:
def indtallest(floor,flats):
    plan=""
    for i in range (floor) :
        plan+=input("please Enter the plan of flats in each floor by typing 0 or 1  :")

    for i in range (0,len(plan)+1,flats):
        test=plan[i:(i+flats)]
        print("[{}]".format(test),'\n')
    tallest=[]


    for i in range(0,floor) :
        c=0
        for j in range (i,len(plan),flats):
            f=int(plan[j])
            c+=f
        tallest.append(c)

    for i in range (len(tallest)):
        tallest[i]=int(tallest[i])

    mx=max(tallest)

    for i in range (len(tallest)):
        if tallest[i] < mx:
            tallest[i]=" "

    print("="*30)
    print(tallest)

#**************************************************    

floor=int(input("please Enter the number of floors :  "))
flats=int(input("please Enter the number of flats in each floor  :  "))
   
indtallest(floor,flats)    

please Enter the number of floors :  3
please Enter the number of flats in each floor  :  3
please Enter the plan of flats in each floor by typing 0 or 1  :100
please Enter the plan of flats in each floor by typing 0 or 1  :110
please Enter the plan of flats in each floor by typing 0 or 1  :101
[100] 

[110] 

[101] 

[] 

[3, ' ', ' ']


please Enter the number of floors :  3
please Enter the number of flats in each floor  :  3
please Enter the plan of flats in each floor by typing 0 or 1  :111
please Enter the plan of flats in each floor by typing 0 or 1  :101
please Enter the plan of flats in each floor by typing 0 or 1  :100
[111] 

[101] 

[100] 

[3, ' ', ' ']



******************************
please Enter the number of floors :  4
please Enter the number of flats in each floor  :  3
please Enter the plan of flats in each floor by typing 0 or 1  :111
please Enter the plan of flats in each floor by typing 0 or 1  :100
please Enter the plan of flats in each floor by typing 0 or 1  :101
please Enter the plan of flats in each floor by typing 0 or 1  :110
[111] 

[100] 

[101] 

[110] 

[] 

==============================
[4, ' ', ' ', ' ']


*************************************

please Enter the number of floors :  4
please Enter the number of flats in each floor  :  4
please Enter the plan of flats in each floor by typing 0 or 1  :0011
please Enter the plan of flats in each floor by typing 0 or 1  :0110
please Enter the plan of flats in each floor by typing 0 or 1  :1010
please Enter the plan of flats in each floor by typing 0 or 1  :1110
[0011] 

[0110] 

[1010] 

[1110] 

[] 

==============================
[' ', ' ', 4, ' ']
